modules/tinymce/src/core/test/ts/browser/html/DomParserTest.ts

```diff
@@ -115,7 +121,7 @@ describe('browser.tinymce.core.html.DomParserTest', () => {
   it('Whitespace preserved in PRE', () => {
     parser = DomParser({}, schema);
     root = parser.parse('  \t\r\n  <PRE>  \t\r\n   test  \t\r\n   </PRE>   \t\r\n  ');
-    assert.equal(serializer.serialize(root), '<pre>  \t\r\n   test  \t\r\n   </pre>', 'Whitespace around and inside PRE');
+    assert.equal(serializer.serialize(root), '<pre>  \t\n   test  \t\n   </pre>', 'Whitespace around and inside PRE');
```

Comment:
Even in whitespace sensitive elements, it turns out `\r` isn't meant to be preserved, so our old parser was incorrect here.